### DISCUSSION TITLE
How to change php version

### DISCUSSION THREAD

**Discussion by nassimb667** [leokhoa/laragon#716] [Laragon's Articles & How-tos]
Hello, i've download php8.3 and try to put it in C:\laragon\bin\php but when i start it i still get some errors can someone help me please? 
Thancks,

**Comment by Pen-y-Fan** [comment]
See #576 PHP 8.3 now available, Questions on which versions are usable with Laragon Full 6.0 220916

**Reply by nassimb667** [reply]
Ok thancks

**Reply by akvariefisk** [reply]
I got the same error and just updated to latest Apache and it works perfect.
